x-pack/plugins/licensing/public/plugin.ts

```diff
@@ -124,11 +123,20 @@ export class LicensingPlugin implements Plugin<LicensingPluginSetup> {
       this.licenseFetchSubscription.unsubscribe();
       this.licenseFetchSubscription = undefined;
     }
+    if (this.storageSubscription !== undefined) {
+      this.storageSubscription.unsubscribe();
+      this.storageSubscription = undefined;
+    }
   }
 
   private fetchLicense = async (core: CoreSetup): Promise<ILicense> => {
     try {
-      const response = await core.http.get(this.infoEndpoint);
+      const response = await core.http.get(this.infoEndpoint, {
+        headers: {
+          'kbn-system-api': 'true',
```

Comment:
right now all calls extend user session. I missed this point in the first PR

Reply:
Not as part of this PR, but what do you think about introducing "system calls" as a first-class setting on the core `http` service? I imagine this is something we'll come across in other apps too, wherever background work is performed not related to user actions

Reply:
@legrego we consider to do this in https://github.com/elastic/kibana/issues/43970